mapper/mapper.go

```diff
@@ -143,6 +149,7 @@ func createPreGTMJavaScript(title string, description model.ReleaseDescription)
 			"release-date-status": "` + description.ProvisionalDate + `",
 			"next-release-date": "` + description.NextRelease + `",
 			"contact-name": "` + description.Contact.Name + `",
+			"tag": "` + censusTag + `"
```

Comment:
For releases that are described as `published` or `cancelled` but not `census`, this will be an empty string
`"tag": ""`
Is that intentional/acceptable?

Reply:
Yes this acceptable for now. At the moment this is a prove of concept and then we can find out to the GTM team is leaving this empty is ok.